src/test/java/spoon/test/model/SwitchCaseTest.java

```diff
@@ -365,4 +367,52 @@ public void test_addCaseAt_throwsIndexOutOfBoundsException_whenPositionIsOutOfBo
 			assertThrows(IndexOutOfBoundsException.class, () -> switchExpression.addCaseAt(3, onlyCase));
 		}
 	}
+	
+	@DisplayName("should return the first case expression")
+	@Test
+	public void test_getCaseExpression_shouldReturnTheFirstCaseExpression() {
+		Factory factory = new Launcher().getFactory();
+
+		CtCase<Integer> ctCase = factory.createCase();
+		CtExpression<Integer> first = factory.createCodeSnippetExpression("1");
+		CtExpression<Integer> second = factory.createCodeSnippetExpression("2");
+		CtExpression<Integer> third = factory.createCodeSnippetExpression("3");
+
+		ctCase.setCaseExpressions(Arrays.asList(first, second, third));
+
+		assertEquals(first, ctCase.getCaseExpression());
```

Comment:
This statement would be more coherent once https://github.com/INRIA/spoon/pull/4024 is merged. As of now, substituting `second` or `third` instead of `first` would still pass the test.